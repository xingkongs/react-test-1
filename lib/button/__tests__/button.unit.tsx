import renderer from "react-test-renderer";
import React from "react";
import Button from "../button";
describe("button", function () {
    it("是一个div", () => {
        const json = renderer.create(<Button value="button"/>).toJSON();
        expect(json).toMatchSnapshot();
    });
});