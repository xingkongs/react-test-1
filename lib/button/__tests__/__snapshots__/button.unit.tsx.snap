// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是一个div 1`] = `
<button
  className="xrui-button"
  onAnimationEnd={[Function]}
  onClick={[Function]}
>
  <span
    className="xrui-button_content"
  >
    button
  </span>
  
</button>
`;
